actions/bump-version/action.yaml

```
@@ -25,20 +41,28 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - name: Set up bump2version
+    - name: Set up bump-my-version
       run: |
-        pipx install bump2version
+        pipx install bump-my-version
       shell: bash
 
     - name: Bump version
       id: bump-version
       run: |
-        parameters=(--no-commit --no-tag ${{ inputs.release-type }})
-        echo "old-version=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
-        if [ -n "${{ inputs.new-version }}" ]; then
-          parameters+=(--new-version ${{ inputs.new-version }})
+        parameters=(--no-commit --allow-dirty --no-tag)
+        echo "old-version=$(bump-my-version show current_version | tail -1)" >> "$GITHUB_OUTPUT"
+
+        if [[ ${{ inputs.new-version }} != "" ]]; then
+          bump-my-version --new-version ${{ inputs.new-version }}
+        else
+          bump-my-version "${parameters[@]}" ${{ inputs.release-type }}
         fi
-        bump2version "${parameters[@]}"
-        echo "new-version=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
+        echo "bump-my-version show current_version | tail -1)" >> "$GITHUB_OUTPUT"
```

I think you are missing `$(`, right? Please test:
```suggestion
        echo "$(bump-my-version show current_version | tail -1)" >> "$GITHUB_OUTPUT"
```